doc/en/components/grids/_shared/filtering.md

```diff
@@ -342,7 +344,58 @@ constructor() {
 }
 ```
 
-<!-- end: Angular, WebComponents -->
+```razor
```

Comment:
Can you add a more complex variant of the filter. For example two columns. I think it would be helpful to see how to construct it, since there isn't a lot of info on that.